src/components/_input.scss

```diff
@@ -30,8 +30,10 @@
     box-sizing: border-box;
     width: auto;
     min-width: none;
-    padding: 0.5em 0.75em;
+    height: 2.25rem;
+    padding: 0 0.75em;
     color: color.$color-grayDarker;
+    line-height: 2.25rem;
```

Comment:
So I was a bit frustrated to find that in inputs, at least in Firefox, it cuts off the descenders with padding. While I don’t want to hard-set heights, maybe it’s OK because by default we set the text size, too? Anyway this fixes the descender cutoff problem.

Reply:
I think that's fine to set here, yeah. It's a consistent size and single line only so the line height doesn't come into play. Text areas though... 🤷